packages/launchpad/src/components/topnav/TopNav.vue

```diff
@@ -0,0 +1,294 @@
+        >
+          Version {{ browser.version }}
+        </div>
+      </div>
+      <template
+        v-if="browser.isSelected"
+        #suffix
+      >
+        <div>
+          <i-cy-circle-check_x24
+            class="icon-dark-jade-100 icon-light-jade-500 w-24px h-24px"
+          />
+        </div>
+      </template>
+    </TopNavListItem>
+  </TopNavList>
+
+  <TopNavList variant="panel">
+    <template #heading="{ open }">
+      <i-cy-life-ring_x16
+        class="icon-dark-gray-500 icon-light-gray-100 group-hocus:icon-dark-indigo-500 group-hocus:icon-light-indigo-50 h-16px w-16px"
+        :class="open ? 'icon-dark-indigo-500 icon-light-indigo-50' : 'icon-dark-gray-500 icon-light-gray-100'"
+      />
+      <span>Docs</span>
+    </template>
+    <div class="flex p-4 gap-24px">
+      <div
+        v-for="list in docsMenu"
+        :key="list.title"
+        class="min-w-164px"
+      >
+        <h2 class="font-semibold text-gray-800">
+          {{ list.title }}
+        </h2>
+        <hr class="border-gray-50 my-2.5">
+        <ul>
+          <li
+            v-for="item in list.children"
+            :key="item.text"
+            class="flex items-center mb-2 text-indigo-500"
+          >
+            <i-cy-book_x16 class="icon-dark-indigo-500 icon-light-indigo-50" />
+            <a
+              :href="getUrl(item.link)"
+              target="_blank"
+              class="ml-2 font-normal whitespace-nowrap"
+            >{{ item.text }}</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </TopNavList>
+</template>
+
+<script setup lang="ts">
+import TopNavListItem from './TopNavListItem.vue'
+import TopNavList from './TopNavList.vue'
+import { allBrowsersIcons } from '../../../../frontend-shared/src/assets/browserLogos'
+import { gql } from '@urql/vue'
+import type { TopNavFragment } from '../../generated/graphql'
+
+const getUrl = (link) => {
+  let result = link.url
+
+  if (link.params) {
+    result += `?${new URLSearchParams(link.params).toString()}`
+  }
+
+  return result
+}
+
+const releasesUrl = 'https://github.com/cypress-io/cypress/releases/'
+
+const utm_medium = 'Docs Menu'
+
+// TODO: will come from gql
+const versionList = [
+  {
+    version: '8.4.1',
+    released: '2 days ago',
+  },
+  {
+    version: '8.4.0',
+    released: '6 days ago',
+  },
+  {
+    version: '8.3.1',
+    released: '12 days ago',
+  },
+  {
+    version: '8.3.0',
+    released: '2 weeks ago',
+  },
+]
+
+gql`
+fragment TopNavContent on App {
+  selectedBrowser {
+    id
+    displayName
+    majorVersion
+  }
+  browsers {
+    id
+    name
+    family
+    disabled
+    isSelected
+    channel
+    displayName
+    path
+    version
+    majorVersion
+  }
+}
+`
+
+const props = defineProps<{
+    gql: TopNavFragment,
+    showBrowsers?: Boolean
+}>()
+
+const docsMenu = [{
+  title: 'Getting Started',
+  children: [{
+    text: 'Write your first test',
```

Comment:
I'd love it if all of the english strings were inside of the en-us json. I use them for tests in addition to giving us a single place to change/update all language.